[Issue: #327] Enable parallel build on C++ extensions (#1265)
intel/scikit-learn-intelex: INSTALL.md

- Install into a newly created environment (recommended):

  ```bash
  conda create -n env -c conda-forge python=3.10 scikit-learn-intelex
  ```

>   If you do not specify the version of Python (`python=3.9` in the example above), then Python 3.11 is downloaded by default.

Reviewer: ```suggestion
  If you do not specify the version of Python (`python=3.10` in the example above), then latest Python is downloaded by default.
```

##### 📦 Supported configurations for Anaconda Cloud from Intel channel <!-- omit in toc -->

| OS / Python version | **Python 3.6** | **Python 3.7** | **Python 3.8** | **Python 3.9** | **Python 3.10** |
| :------------------ | :------------: | :------------: | :------------: | :------------: | :------------:  |
| **Linux**           |   [CPU, GPU]   |   [CPU, GPU]   |   [CPU, GPU]   |   [CPU, GPU]   |   [CPU, GPU]    |
| **Windows**         |   [CPU, GPU]   |   [CPU, GPU]   |   [CPU, GPU]   |   [CPU, GPU]   |   [CPU, GPU]    |
| **MacOS**           |     [CPU]      |     [CPU]      |     [CPU]      |     [CPU]      |     [CPU]       |

#### Install via Anaconda Cloud from Main channel

- Install into a newly created environment (recommended):

  ```bash
  conda create -n env python=3.10 scikit-learn-intelex
  ```

>   If you do not specify the version of Python (`python=3.10` in the example above), then latest Python is downloaded by default.

Reviewer 2: ```suggestion
..note:  If you do not specify the version of Python, the latest one is downloaded. 
```

>   conda create -n env python=3.9 scikit-learn-intelex

Reviewer: ```suggestion
  conda create -n env python=3.10 scikit-learn-intelex
```

  ```

>   If you do not specify the version of Python (`python=3.9` in the example above), then Python 3.10 is downloaded by default.

Reviewer: ```suggestion
  If you do not specify the version of Python (`python=3.10` in the example above), then latest Python is downloaded by default.
```
